scalafmt-core/shared/src/main/scala/org/scalafmt/util/TokenTraverser.scala

```diff
@@ -42,25 +42,45 @@ class TokenTraverser(tokens: Tokens) {
     }
   }
 
+  def findAfter(
```

Comment:
Can you add some explanation about the behavior of `predicate` with scaladoc?: it's a bit difficult to catch the meaning of this method for me.

- If it returns None: skip and keep searching.
- Some(false): stop searching.
- Some(true): return that token.